src/main/service/FirebaseConfig.ts

```diff
@@ -0,0 +1,15 @@
+import * as admin from 'firebase-admin';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export function initFirebaseAdmin() {
+  admin.initializeApp({
+    serviceAccountId: process.env.FIREBASE_SERVICE_ACCOUNT_ID,
+    credential: admin.credential.cert({
+      projectId: process.env.FIREBASE_PROJECT_ID,
+      privateKey: process.env.FIREBASE_PRIVATE_KEY,
+      clientEmail: process.env.FIREBASE_CLIENT_EMAIL,
```

Comment:
Also, could we add all these new environment variables to `sample.env` and update the read me too?

Reply:
Hmm, I think we could not since the web api key and private key of firebase must be secret.

Reply:
Oh I meant this file here https://github.com/ubclaunchpad/PromoPal-backend/blob/user-endpoints/sample.env. That way future devs know which api keys they need to copy. You don't need to put the actual value of the key, u can just set the value as `INSERT_HERE`.

Reply:
ahh, I see lol. I'll put some needed variables.